### PR TITLE
took out config fallback to martha_v2 [WA-352]

### PR DESCRIPTION
**Pull request by rsasch**
Closes https://broadworkbench.atlassian.net/browse/WA-352

### CODE REVIEW

**Comment by rsasch** [comment]
jenkins retest

File: core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala [hunk 1]
```diff
@@ -264,9 +264,8 @@ object Boot extends IOApp with LazyLogging {
         gcsConfig.getString("notifications.topicName")
       )
 
-      val marthaConfig: Config = conf.getConfig("martha")
-      val marthaBaseUrl: Option[String] = marthaConfig.getStringOption("baseUrl")
-      val marthaUrl = marthaBaseUrl.map(_ + "/martha_v3").getOrElse(marthaConfig.getString("baseUrl_v2"))
+      val marthaBaseUrl: String = conf.getConfig("martha").getString("baseUrl")
+      val marthaUrl: String = marthaBaseUrl.flatMap(_ + "/martha_v3")
```
Comment (salonishah11) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala]: flatMapping on a String would split the string into characters and append `/martha_v3` at the end of each character. 
So for example:
```
val strVal = "hello"
val path = strVal.flatMap(_ + "/world")
println(path)
```
will result in:
```
h/worlde/worldl/worldl/worldo/world
```

Instead one can do 
```
val marthaUrl: String = s"$marthaBaseUrl/martha_v3"
```
Code:
```
val strVal = "hello"
val path = s"$strVal/world"
println(path)
```
will result in:
```
hello/world
```

**Review by salonishah11 — CHANGES_REQUESTED** [review]

**Review by kshakir — COMMENTED** [review]
Concur with Saloni. Re-ping me here when ready for re-review.

File: core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala (hunk 1 above)
Comment (rsasch) [reply core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala]: Thank you!  Done.

File: core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala [hunk 2]
```diff
@@ -264,9 +264,8 @@ object Boot extends IOApp with LazyLogging {
         gcsConfig.getString("notifications.topicName")
       )
 
-      val marthaConfig: Config = conf.getConfig("martha")
-      val marthaBaseUrl: Option[String] = marthaConfig.getStringOption("baseUrl")
-      val marthaUrl = marthaBaseUrl.map(_ + "/martha_v3").getOrElse(marthaConfig.getString("baseUrl_v2"))
+      val marthaBaseUrl: String = conf.getConfig("martha").getString("baseUrl")
```
Comment (kshakir) [inline core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala]:
```suggestion
      val marthaBaseUrl: String = conf.getString("martha.baseUrl")
```

**Review by kshakir — APPROVED** [review]

**Review by salonishah11 — APPROVED** [review]

**Pull request merged**